### DISCUSSION TITLE
How to add field to admin panel users

### DISCUSSION THREAD

**Discussion by marcusprice** [strapi/strapi#7943] [Legacy Topics]
Hello! I am building a blog for someone w/strapi and would like to add an additional field to the user's type called "penName". This is so they have control of how their name appears on the frontend of the blog while still keeping their first and last names as part of the model. I have a couple of questions:

1. How do I add a field to the users type that has access to the strapi admin panel (found under the Administration Panel section in settings)? There seems to be no way to add to or edit that type in the admin panel.
2. There is a pre-made Users content type when you do a fresh install of strapi. Why can't users that are created there log into the admin panel? What is this Users content type for? Is it for 3rd party users that might want access to that api but need to be authenticated first?

Sorry if these are dumb questions or if I missed something. I've searched through the docs and online and haven't found anything. Thanks in advance!

**Comment by derrickmehaffy** [comment]
Hi @marcusprice currently it's not possible (though we will be allowing for `oneWay` and `manyWay` relations from the admin user model in a future version).

See the following issue as to why: https://github.com/strapi/strapi/issues/4838

**Reply by marcusprice** [reply]
Dang, okay. Thanks for letting me know. Is there still no ETA on this?

**Reply by derrickmehaffy** [reply]
No ETA yet no.

**Reply by soupette** [reply]
The other way would be for you to fork the `strapi-admin` package and install your fork in your project with the added fields instead of the official one.
